manifests/apache.pp

```diff
@@ -1,101 +1,46 @@
 # Setups Apache for Katello development
 class katello_devel::apache {
 
-  include apache
+  include foreman::params
+
+  class { 'foreman::config::apache':
+    passenger               => false,
+    ssl                     => true,
+    ssl_cert                => $certs::apache::apache_cert,
+    ssl_key                 => $certs::apache::apache_key,
+    ssl_ca                  => $certs::apache::ca_cert,
+    ssl_chain               => $certs::apache::ca_cert,
+		app_root                => '/home/vagrant/foreman',
+		passenger_ruby          => $foreman::params::passenger_ruby,
+		priority                => $foreman::params::vhost_priority,
+		servername              => $foreman::params::servername,
+		serveraliases           => $foreman::params::serveraliases,
+		server_port             => $foreman::params::server_port,
+		server_ssl_port         => $foreman::params::server_ssl_port,
+		ssl_certs_dir           => '',
+		ssl_crl                 => '',
+		user                    => $foreman::params::user,
+		passenger_prestart      => false,
+		passenger_min_instances => 0,
+		passenger_start_timeout => 0,
+		foreman_url             => $foreman::params::foreman_url,
+		ipa_authentication      => false,
+    proxy_backend           => "http://0.0.0.0:${katello_devel::rails_port}/",
+    ssl_protocol            => undef,
+    ssl_verify_client       => 'optional',
+  }
 
-  $priority = '05'
+  foreman::config::apache::fragment { 'katello':
+    ssl_content => file('katello/katello-apache-ssl.conf'),
+  }
 
   $proxy_pass_https = [
```

Comment:
@ekohl I wasn't sure how to declare this additional proxy pass information to the class.

Reply:
I actually wonder if this is really needed since I'm not sure what listens on port 6006. Looks like the storybook according to 3a61896824f67b050f4ce31467adea1e1f7e5b7c but perhaps the devs should just go to `http://$hostname:6006` for that.